docs/manual/working/scalaGuide/main/json/ScalaJson.md

```diff
@@ -101,22 +101,26 @@ You can traverse a `JsValue` structure and extract specific values. The syntax a
 
 ### Simple path `\`
 
-Applying the `\` operator to a `JsValue` will return the property corresponding to the field argument, supposing this is a `JsObject`.
+Applying the `\` operator to a `JsValue` will return the property corresponding to the field argument in a `JsObject`, or the item at that index in a `JsArray`
 
 @[traverse-simple-path](code/ScalaJsonSpec.scala)
 
+In this example we are using `.get`, which retrieves the value out of the resulting `JsResult` and throws an exception if it was not found. You can also use the `JsResult` directly to exceptions, or use the Direct lookup `.apply` method below if you want a missing key or index to throw.
```

Comment:
Here's a suggestion to clarify (note that `JsResult` is the result of parsing JSON, while `JsLookupResult` is the result of a lookup):

The `\` operator returns a `JsLookupResult`, which is either `JsDefined` or `JsUndefined`. You can chain multiple `\` operators, and the result will be `JsUndefined` if any intermediate value cannot be found. Calling `get` on a `JsLookupResult` attempts to get the value if it is defined and throws an exception if it does not exist.

You can also use the `apply` method to get a field in an object or index in an array. Like `get`, this method will throw an exception if the value does not exist.